src/v2/Components/Inquiry/views/InquiryBasicInfo.tsx

```diff
@@ -1,19 +1,215 @@
+import {
+  Banner,
+  Box,
+  Button,
+  Checkbox,
+  Input,
+  Skeleton,
+  SkeletonBox,
+  SkeletonText,
+  Text,
+} from "@artsy/palette"
 import React from "react"
-import { Button, Text } from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+import { SystemQueryRenderer } from "v2/System/Relay/SystemQueryRenderer"
 import { useInquiryContext } from "../InquiryContext"
+import { InquiryBasicInfo_artwork } from "v2/__generated__/InquiryBasicInfo_artwork.graphql"
+import { InquiryBasicInfoQuery } from "v2/__generated__/InquiryBasicInfoQuery.graphql"
+import { useSystemContext } from "v2/System"
+import {
+  Location,
+  LocationAutocompleteInput,
+  normalizePlace,
+  Place,
+} from "v2/Components/LocationAutocompleteInput"
+import { useState } from "react"
+import { useUpdateMyUserProfile } from "../useUpdateMyUserProfile"
 
-export const InquiryBasicInfo: React.FC = () => {
+enum Mode {
+  Pending,
+  Loading,
+  Success,
+  Error,
+}
+
+interface InquiryBasicInfoProps {
+  artwork: InquiryBasicInfo_artwork
+}
+
+const InquiryBasicInfo: React.FC<InquiryBasicInfoProps> = ({ artwork }) => {
   const { next } = useInquiryContext()
+  const [mode, setMode] = useState(Mode.Pending)
+  const { submitUpdateMyUserProfile } = useUpdateMyUserProfile()
+
+  const [state, setState] = useState<{
+    profession: string | null
+    location: Location | null
+    phoneNumber: string | null
+    shareFollows: boolean
+  }>({
+    profession: null,
+    location: null,
+    phoneNumber: null,
+    shareFollows: false,
+  })
+
+  const handleLocation = (place: Place) => {
+    setState(prevState => ({ ...prevState, location: normalizePlace(place) }))
+  }
+
+  const handleSelect = (value: boolean) => {
+    setState(prevState => ({ ...prevState, shareFollows: value }))
+  }
+
+  const handleInputChange = (name: "profession" | "phoneNumber") => (
+    event: React.ChangeEvent<HTMLInputElement>
+  ) => {
+    setState(prevState => ({ ...prevState, [name]: event.target.value }))
+  }
+
+  const handleSubmit = async (event: React.FormEvent<HTMLElement>) => {
+    event.preventDefault()
+
+    setMode(Mode.Loading)
+
+    try {
+      await submitUpdateMyUserProfile({ phone: state.phoneNumber })
+      setMode(Mode.Success)
+      next()
+    } catch (err) {
+      console.error(err)
```

Comment:
If possible can we use `logger`? Then, in the absence of perfect sentry sourcemaps we get the file name where the error is originating: https://github.com/artsy/force/blob/master/src/v2/System/Router/buildClientApp.tsx#L36

Reply:
Ah yeah, I didn't even know this was a thing. Nice.